shared-data/python/tests/protocol/test_protocol_schema_v6.py

```diff
@@ -0,0 +1,12 @@
+import json
+import pytest
+from opentrons_shared_data import load_shared_data
+from opentrons_shared_data.protocol.models import ProtocolSchemaV6
+
+from . import list_fixtures
+
+
+@pytest.mark.parametrize("defpath", list_fixtures(6))
+def test_v6_types(defpath):
+    json.loads(load_shared_data(defpath))
+    ProtocolSchemaV6.parse_raw(load_shared_data(defpath))
```

Comment:
Let's make sure we're actually checking our model parsing a little here:

```suggestion
    def_data = load_shared_data(defpath)

    def_dict = json.loads(def_data)
    def_model = ProtocolSchemaV6.parse_raw(def_data)
    def_dict_from_model = def_model.dict(exclude_none=True)

    # pop the schema ID off the dicts so we can compare the full dicts later
    expected_schema = def_dict.pop("$otSharedSchema")
    actual_schema = def_dict_from_model.pop("otSharedSchema")

    assert actual_schema == expected_schema
    assert def_dict_from_model == def_dict
```